src/main/java/NovaGPT/command/TaskList.java

```diff
@@ -0,0 +1,122 @@
+package NovaGPT.command;
+
+import NovaGPT.task.Task;
+import NovaGPT.task.Todo;
+import NovaGPT.task.Deadline;
+import NovaGPT.task.Event;
+import NovaGPT.ui.Ui;
+import NovaGPT.storage.Storage;
+import NovaGPT.exception.NovaException;
+import java.util.ArrayList;
+
+public class TaskList {
+    private static final String MARKED_MESSAGE = "Nice! I've marked this task as done:";
+    private static final String UNMARKED_MESSAGE = "OK, I've marked this task as not done yet:";
+
+    public static void handleTodoTask(String input, ArrayList<Task> ls, Storage st) {
+        String text = input.substring(4).trim();
+        if (text.isEmpty()) {
+            Ui.print("OOPS! The description of a todo cannot be empty. " +
+                    "\nDo format your input: todo <task>");
+            return;
+        }
+        Task t = new Todo(text);
+        ls.add(t);
+        st.save(ls);
+        Ui.print("Got it. I've added this task:\n"
+                + t.toString() + "\nNow you have "
+                + ls.size()
+                + " tasks in the list.");
```

Comment:
Perhaps you may want to line break after every "\n" character instead?
```suggestion
        Ui.print("Got it. I've added this task:\n"
                + t.toString() + "\n"
                + "Now you have " + ls.size() + " tasks in the list.");
```
This may help to make your code layout similar to the expected output